src/features/people/PersonPage/MainPersonTile/index.js

```diff
@@ -11,23 +11,27 @@ import {
 } from "./styled";
 import personPhoto from "../../../../common/img/person.png";
 
-export const MainPersonTile = () => {
+export const MainPersonTile = ({ poster, name, birthday, birthplace, biography }) => {
     return (
-            <PersonTileWrapper>
-                <PersonPhoto src={personPhoto} />
-                <PersonData>
-                    <PersonName>Liu Yifei</PersonName>
-                    <BasicInfo>
-                        <Birth>Date of birth: </Birth>
-                        <BirthDetails>25.08.1987 </BirthDetails>
-                    </BasicInfo>
-                    <BasicInfo>
-                        <Birth>Place of birth: </Birth>
-                        <BirthDetails>Wuhan, Hubei, China </BirthDetails>
-                    </BasicInfo>
-                </PersonData>
-                <Biography>Liu Yifei was born in Wuhan, Hubei, Province of China on August 25th, 1987. She began modeling at the age of 8 and was trained in singing, dancing and the piano. Moving to the United States at 10 with her mother, Liu lived there for four years.</Biography>
-            </PersonTileWrapper>
+        <PersonTileWrapper>
+            {poster && <PersonPhoto src={`${img}${poster}`} />}
```

Comment:
The code looks nice, but after looking at line 17, I think it's worth changing the name of the constant {img} in the apiCodes.js file to something else, e.g. {IMG_URL) because at first I was wondering if two images were added here by mistake, which in my opinion would not make sense :) That's why I propose a slight correction of the naming in the apiCodes.js file. What do you think? :) @BoAneta @Dawid-Zieba

Reply:
It's good idea to change img in apiCodes for Img_url

Reply:
Good idea. Im gonna changed it